internal/qoscfg/qoscfg.go

```diff
@@ -32,6 +32,16 @@ func SetForwardingGroup(t *testing.T, dut *ondatra.DUTDevice, qos *oc.Qos, group
 	gnmi.Replace(t, dut, gnmi.OC().Qos().Config(), qos)
 }
 
+func SetForwardingGroupWithFabricPriority(t *testing.T, dut *ondatra.DUTDevice, qos *oc.Qos, groupName, queueName string, fabricPriority uint8) {
```

Comment:
This func should have a comment documenting its purpose.
e.g.,
// SetForwardingGroupWithFabricPriority sets a forwarding group with fabric priority for a given queue in the specified qos config

(please correct as appropriate)

Reply:
Removed this function from the `qoscfg` library. Considering, it is not required after fabric-priority configurations are removed as discussed in b/297188157.